src/components/OrderItem/OrderItem.jsx

```diff
@@ -1,16 +1,16 @@
-import React from "react";
-import { Price } from "../../components";
+import React from 'react';
+import { Price } from '../../components';
 
 export class OrderItem extends React.Component {
```

Comment:
Both this component `OrderItem` and `Order` can be refactored into functional components, they don't need to extend `React.Component`. They are just passing the function down from the parent - no need to redefine it as a new function called `action` man. You can destructure in the statement too so it can be an implicit return.

```
export const OrderItem = ( { id, removeFromOrder } ) => (
```